eng/pipelines/aggregate-reports.yml

```diff
@@ -1,6 +1,67 @@
 trigger: none
 pr: none
 
-steps:
-    - script: | 
-        hello world
\ No newline at end of file
+
+jobs:
+  - job: Generate
+    variables:
+      - template: templates/variables/globals.yml
+    pool:
+      vmImage: 'ubuntu-16.04'
+    steps:
+    - template: templates/steps/install-reporting-tools.yml
+      parameters:
+        Options: --batch-mode
+        MavenOptions: $(MemoryOptions) $(LoggingOptions)
+
+    - task: Maven@3
+      displayName: 'Install azure-sdk-parent'
+      inputs:
+        mavenPomFile: parent/pom.xml
+        options: '$(DefaultOptions) -DskipTests -Dgpg.skip'
+        mavenOptions: '$(MemoryOptions)  $(LoggingOptions)'
+        javaHomeOption: 'JDKVersion'
+        jdkVersionOption: '1.11'
+        jdkArchitectureOption: 'x64'
+        publishJUnitResults: false
+        goals: 'install'
+
+    - task: Maven@3
+      displayName: 'Generate Maven project site, including JavaDocs, SpotBugs, and CheckStyle reports'
+      inputs:
+        mavenPomFile: pom.client.xml
+        options: '$(DefaultOptions) -DskipTests -Dgpg.skip'
+        mavenOptions: '$(MemoryOptions) $(LoggingOptions)'
+        javaHomeOption: 'JDKVersion'
+        jdkVersionOption: '1.11'
+        jdkArchitectureOption: 'x64'
+        publishJUnitResults: false
+        goals: 'install site:site site:stage'
```

Comment:
Don't need `install` here. `site:site site:stage` will generate all 3 reports.

Reply:
Seems we do need it after some experimentation :-)